src/System.Net.HttpListener/src/System/Net/Managed/HttpEndPointListener.cs

```diff
@@ -56,7 +56,11 @@ public HttpEndPointListener(HttpListener listener, IPAddress addr, int port, boo
             {
                 _secure = secure;
                 // TODO #14691: Implement functionality to read SSL certificate.
+#if MONO
+                _cert = _listener.LoadCertificateAndKey (addr, port);
```

Comment:
If for some reason we can't, I'd like us not to use ifdefs.  We can move the relevant functionality into partial classes, and Mono can supply its own implementation of those partials.

But I'd really like to see if we could have a single implementation used by both corefx and Mono, porting whatever we need to from Mono to make this work, and work consistently for everyone.